tests/autobahn/server/server.py

```diff
@@ -30,29 +29,16 @@ async def wshandler(request):
     return ws
 
 
-async def main(loop):
+def main():
     app = web.Application()
     app.router.add_route("GET", "/", wshandler)
 
-    handler = app._make_handler()
-    srv = await loop.create_server(handler, "127.0.0.1", 9001)
-    print("Server started at http://127.0.0.1:9001")
-    return app, srv, handler
-
-
-async def finish(app, srv, handler):
-    srv.close()
-    await handler.shutdown()
-    await srv.wait_closed()
+    web.run_app(app, port=9001)
+    print("Server started at http://0.0.0.0:9001")
```

Comment:
This is misleading. `web.run_app()` blocks and `print()` is not executed after the server start but after the server shutdown.